File: Cliente_proyecto_final/main/java/com/example/practica3/Datos/Usuario.java
package com.example.practica3.Datos;

import java.io.Serializable;

public class Usuario implements Serializable {

    public int id;
    public String nombre;
    public String apellidos;
    public String email;
    public String contrasenia;

    public Usuario(int id, String nombre, String apellidos, String email, String contrasenia) {
        this.id = id;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.email = email;
        this.contrasenia = contrasenia;
    }
}

File: Servidor_proyecto_final/DataObjects/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace servidorProyectoComercioElecronico.DataObjects
{
    public class Usuario
    {
        public int id { get; set; }
        public string nombre { get; set; }
        public string apellidos { get; set; }
        public string email { get; set; }
        public string contrasenia { get; set; }
    }
}